packages/flutter_tools/bin/podhelper.rb

```diff
@@ -73,9 +73,13 @@ def flutter_additional_ios_build_settings(target)
 
 # Same as flutter_ios_podfile_setup for macOS.
 def flutter_additional_macos_build_settings(target)
-  print target.platform_name
   return unless target.platform_name == :osx
 
+  # [target.deployment_target] is a [String] formatted as "10.8". Check if this is < 10.9
```

Comment:
To make the source of these numbers as clear as possible to future readers/maintainers, how about "Check if this is less than the minimum OS version supported by the latest stable version of Xcode (currently 10.9)"

Reply:
The details are in the `See MACOSX_DEPLOYMENT_TARGET note below.` larger comment.  I'll move it up.